filebeat/tests/system/test_shutdown.py

```diff
@@ -43,27 +42,31 @@ def test_shutdown_wait_ok(self):
 
         # Wait until first flush
         self.wait_until(
-            lambda: self.log_contains_count("Flushing spooler") > 1,
+            lambda: self.log_contains_count("Publish event") > 200,
```

Comment:
before it was checking for 'Flushing spooler' message. which can be any number of events (up to spooler size). As spooler and any kind of flushing is gone, I have had to use a replacement on number of events being published. The test file contains like 50k events. 200 should be ok, by we can decrease it.